tox.ini

```diff
@@ -33,10 +33,6 @@ deps =
     pip >= 19.3.1
 extras = testing
 commands = pytest \
-           --cov "{envsitepackagesdir}/tox" \
```

Comment:
Eh what?

Reply:
Just speeding up my testing whilst developing.  Needs removing before un-drafting this.